contracts/messages.tact

```diff
@@ -4,7 +4,9 @@ message(0x133701) JettonInit {
     jetton_description: Slice;
     jetton_symbol: Slice;
     max_supply: Int as coins;
+    mint_amount: Int? as coins;
 }
+
```

Comment:
I think we should set up prettier for project. Because my vscode plugin for tact deletes these line breaks automatically. Not related to this issue. Just thoughts. I will open issue for it